spec/zoneddatetime.html

```diff
@@ -1187,13 +1222,26 @@ <h1>CreateTemporalZonedDateTime ( _epochNanoseconds_, _timeZone_, _calendar_ [ ,
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-temporalzoneddatetimetostring" aoid="TemporalZonedDateTimeToString">
-      <h1>TemporalZonedDateTimeToString ( _zonedDateTime_, _precision_, _showCalendar_, _showTimeZone_, _showOffset_ [ , _increment_, _unit_, _roundingMode_ ] )</h1>
-      <p>
-        The abstract operation TemporalZonedDateTimeToString returns an ISO 8601 string representation of its argument, including a time zone name annotation and calendar annotation, which are extensions to the ISO 8601 format.
-      </p>
+    <emu-clause id="sec-temporal-temporalzoneddatetimetostring" type="abstract operation">
+      <h1>
+        TemporalZonedDateTimeToString (
+          _zonedDateTime_: a Temporal.ZonedDateTime,
+          _precision_: a String or integer,
+          _showCalendar_: a String,
+          _showTimeZone_: a String,
+          _showOffset_: a String,
+          optional _increment_: an integer,
+          optional _unit_: a String,
+          optional _roundingMode_: a String,
+        )
```

Comment:
```suggestion
        ): a String
```

Reply:
When I do this, I get a bunch of errors:
```
Error: spec/zoneddatetime.html:1252:38: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1253:36: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1257:32: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1262:34: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:1264:32: this would return a Completion Record, but the containing AO is declared not to return a Completion Record
Error: spec/zoneddatetime.html:839:1: TemporalZonedDateTimeToString does not return a Completion Record, but is consumed as if it does
Error: spec/zoneddatetime.html:855:1: TemporalZonedDateTimeToString does not return a Completion Record, but is consumed as if it does
Error: spec/zoneddatetime.html:867:1: TemporalZonedDateTimeToString does not return a Completion Record, but is consumed as if it does
```

Reply:
ah, in that case:
```suggestion
        ): a normal completion containing a String
```

Reply:
Done :+1: